templates/csharp/manifest.yaml

```diff
@@ -0,0 +1,5 @@
+# Optional list of additional Buildpacks to be applied to the language pack's
+# builder image when the Function is built using a Buildpack builder.
```

Comment:
Please add the following to this file in your language pack and see if that will work for you.

```
builderImages:
    pack: gcr.io/paketo-buildpacks/builder:base
```

Reply:
@clibequilibrium were you able to resolve your issue by using the above suggestion?

Reply:
> @clibequilibrium were you able to resolve your issue by using the above suggestion?

I will get back to you today. Thank you for follow up

Reply:
@lance  is it an oversight from docs? Because all of this seems Optional, shouldn't func still use default builderImage as base one if it's not provided in manifest.yaml?

Reply:
`===> DETECTING
ERROR: No buildpack groups passed detection.
ERROR: Please check that you are running against the correct path.
ERROR: failed to detect: no buildpacks participating
-
Error: failed to build the function: executing lifecycle: failed with status code: 20`

https://github.com/clibequilibrium/func-templates

Reply:
I can successfully create a new project using your repo but it won't build, giving the same error you show above. This error is coming from the paketo builder, saying that it can't find a suitable project to build. I'm not familiar with what kinds of files the paketo builder looks for in order to build a CSharp project, but this is typically the kind of error you would see if it can't find what it's looking for. A good way to test this is to try and build the project using the `pack` CLI and not use `func` at all. Doing that, I see this same error.

https://app.warp.dev/block/aWOnPASSvuuQ31nhNB7KTR

You'll need to add whatever file(s) necessary to indicate to the Paketo builder that this is a csharp project. Also, as an aside - I think you don't need both of these lines - only one, and perhaps neither. I believe this buildpack is included in the Paketo builder, so in that case it should not be necessary to add it in manifest.yaml - but I'm not sure.

https://github.com/clibequilibrium/func-templates/blob/eaa195b57e5cbeaa7447a07b32c2692738fbf35d/manifest.yaml#L13-L15